schema.sql

```diff
@@ -63,7 +64,12 @@ create table fileruns (
 drop table if exists access;
 create table access (
   access_id integer primary key autoincrement,
-  site_access_code varchar not null,
+  server_id varchar not null,
+  server_name varchar,
+  server_address varchar,
+  user_id varchar not null,
+  user_name varchar,
+  user_email varchar,
```

Comment:
I'd suggest moving `user_name` and `user_email` to a `users` table, where the combination (server_id, user_id) is unique.